src/global/reco/reco.cc

```diff
@@ -310,15 +312,42 @@ void InitPlugin(JApplication *app) {
             app
     ));
     app->Add(new JOmniFactoryGeneratorT<FarForwardNeutronReconstruction_factory>(
-           "ReconstructedFarForwardZDCNeutrons",
+           "ReconstructedFarForwardZDCNeutronsStandAlone",
            {"HcalFarForwardZDCClusters","EcalFarForwardZDCClusters"},  // edm4eic::ClusterCollection
-          {"ReconstructedFarForwardZDCNeutrons"}, // edm4eic::ReconstrutedParticleCollection,
+          {"ReconstructedFarForwardZDCNeutronsStandAlone"}, // edm4eic::ReconstrutedParticleCollection,
           {
             .scale_corr_coeff_hcal={-0.0756, -1.91, 2.30},
             .scale_corr_coeff_ecal={-0.352, -1.34, 1.61}
           },
+          app
+    ));
+    app->Add(new JOmniFactoryGeneratorT<FarForwardNeutralsReconstruction_factory>(
+           "ReconstructedFarForwardZDCNeutrons",
+           {"HcalFarForwardZDCClusters"},  // edm4eic::ClusterCollection
+           {"ReconstructedFarForwardZDCNeutrons", "ReconstructedFarForwardZDCGammas"}, // edm4eic::ReconstrutedParticleCollection,
+           {
+            .n_scale_corr_coeff_hcal={-0.11, -1.5, 0},
+            .gamma_scale_corr_coeff_hcal={0, -.13, 0},
+            .rot_y=-0.025,
+            .gamma_zmax=(300+35800)*dd4hep::mm,
```

Comment:
```suggestion
            .gamma_zmax=(300+35800)*dd4hep::mm,
```
Is it possible to retrieve these from the geometry?